frontend/app/mocks/node.ts

```diff
@@ -216,15 +262,59 @@ const handlers = [
       responseType: 'CA',
       statusCode: 'Valid',
       functionalMessages: [
-        {action: 'OriginalInput',
-        message: '111 WELLINGTON ST   OTTAWA   ON   K1A0A4   CAN'},
-        {action: 'Information',
-        message: 'Dept = SENAT   Branch = SENAT   Lang = F'}
+        { action: 'OriginalInput', message: '111 WELLINGTON ST   OTTAWA   ON   K1A0A4   CAN' },
+        { action: 'Information', message: 'Dept = SENAT   Branch = SENAT   Lang = F' },
       ],
       message: '',
       warnings: '',
     });
   }),
+
+  /**
+   * Handler for GET requests to retrieve letters details.
+   */
+  http.get('https://api.example.com/letters?userId=', ({ params }) => {
+    const letterEntities = getLetterEntities(params.userId);
+
+    return HttpResponse.json(
+      letterEntities?.map((letter) => {
+        return {
+          dateSent: letter.dateSent,
+          subject: letter.letterTypeCd,
+          referenceId: letter.referenceId,
+        };
+      }),
+    );
+  }),
+
+  /**
+   * Handler for GET requests to retrieve pdf
+   */
+
+  /**
+   *
+   */
```

Comment:
This can be removed.

Reply:
FIxed: c20e06602530ac9a3fb8cf2b080a3b3fc36d0451.